CMakeLists.txt

```diff
@@ -80,7 +81,7 @@ target_include_directories(${PackageName}G4 PRIVATE ${PROJECT_SOURCE_DIR}/detect
 target_include_directories(${PackageName}   SYSTEM PUBLIC ${LCIO_INCLUDE_DIRS})
 target_include_directories(${PackageName}G4 SYSTEM PUBLIC ${LCIO_INCLUDE_DIRS})
 
-target_link_libraries(${PackageName}   DD4hep::DDCore DD4hep::DDRec DD4hep::DDParsers ${ROOT_LIBRARIES} ${LCIO_LIBRARIES})
+target_link_libraries(${PackageName}   DD4hep::DDCore DD4hep::DDRec DD4hep::DDParsers ${ROOT_LIBRARIES} ${LCIO_LIBRARIES} detectorSegmentations)
```

Comment:
The file `detector/tracker/parametrised_DriftChamber.cpp` requires a segmentation class: `detectorSegmentations/GridSimplifiedDriftChamber_k4geo.h` 

When I tried to include just the header instead of linking, like:
```
target_include_directories(${PackageName}   PRIVATE ${PROJECT_SOURCE_DIR}/detectorSegmentations/include )

target_link_libraries(${PackageName}   DD4hep::DDCore DD4hep::DDRec DD4hep::DDParsers ${ROOT_LIBRARIES} ${LCIO_LIBRARIES} )

```
, it raises an error at the end of the compilation,
```
/cvmfs/sw.hsf.org/contrib/binutils/2.38/x86_64-centos7-gcc8.3.0-opt/pv4x2/bin/ld: CMakeFiles/lcgeo.dir/detector/tracker/parametrised_DriftChamber.cpp.o: in function `det::createParamSimplifiedDriftChamber(dd4hep::Detector&, dd4hep::xml::Handle_t, dd4hep::SensitiveDetector)':
parametrised_DriftChamber.cpp:(.text+0x466): undefined reference to `typeinfo for dd4hep::DDSegmentation::GridSimplifiedDriftChamber_k4geo'
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/lcgeo.dir/build.make:2057: lib/liblcgeo.so] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:158: CMakeFiles/lcgeo.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
```

Reply:
Ok, then it is fine to leave it in.